pkg/products/amqonline/reconciler.go

```diff
@@ -312,15 +312,14 @@ func (r *Reconciler) reconcileStandardAuthenticationService(ctx context.Context,
 			return nil
 		},
 	)
+	if err != nil {
+		return integreatlyv1alpha1.PhaseFailed, fmt.Errorf("could not create postgresql for standard auth service: %w", err)
```

Comment:
@Rajagopalan-Ranganathan with this change, I'm interested to know what the error returned from the `croUtil.ReconcilePostgres` function was. Have you managed to hit that since you made the change?

Reply:
@grdryn - I have not reproduced the issue since the first time. I will keep working on it and let you know if i manage to reproduce it.